### ISSUE TITLE
 "Attribute itemprop not allowed on element link" but should

### ISSUE THREAD

**Issue by 1jj** [validator/validator#893]
"_Attribute itemprop not allowed on element link at this point_"
I get this errors for things like that:
`<link rel=icon itemprop=logo href=ico.png>`
`<link rel=canonical itemprop=url href="https://example.org/">`

Otherwise the spec says
"_Every HTML element may have an itemprop attribute specified_"
https://html.spec.whatwg.org/multipage/microdata.html#names:-the-itemprop-attribute

Google's Structured Data Testing Tool is also fine with it.

**Comment by sideshowbarker** [comment]
Where is the element with an `itemscope` attribute that those elements are associated with? Please post a larger snippet of the document (or else the URL for the document, or the entire document) that shows the microdata structure. The snippets in the document are not enough to determine why the checker is reporting an error.

**Comment by 1jj** [comment]
```
<!DOCTYPE html>
<html>
<head itemscope itemtype=https://schema.org/Organization>
<meta charset=utf-8>
<title itemprop=name>Example Organization</title>
<link itemprop=url rel=canonical href=https://example.org/>
<link itemprop=logo rel=icon href=ico.png>
<link itemprop=image rel=apple-touch-icon href=image.png>
<meta itemprop=telephone content=1234567890>
```
The Google tool gives ok with this code snippet
https://search.google.com/structured-data/testing-tool

Should also work with `<html itemscope itemtype>`

**Comment by sideshowbarker** [comment]
Either remove the `rel` attribute from those `link` elements, or remove the `itemprop` attribute:

https://html.spec.whatwg.org/multipage/semantics.html#the-link-element:names:-the-itemprop-attribute
> A `link` element must have either a `rel` attribute or an `itemprop` attribute, but not both.

Essentially the same issue for the other markup snippet:

> Edit - also "_Attribute itemprop not allowed on element meta_"
> `<meta name=description itemprop=description content=...>`

Either remove the `name` attribute, or remove the `itemprop` attribute:
https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element:names:-the-itemprop-attribute-5
> Exactly one of the `name`, `http-equiv`, `charset`, and `itemprop` attributes must be specified.

**Comment by 1jj** [comment]
Thanks!
> Exactly one of the `name`, `http-equiv`, `charset`, and `itemprop` attributes must be specified.

Such an error message would be helpful then ...

**Comment by sideshowbarker** [comment]
> Such an error message would be helpful then ...

Good point, and agreed — I’ve labeled it and will keep it open

**Comment by earthtopomaps** [comment]
I have a problem with this:
<a href="" itemprop="item">
I tried several entries like this:
-------------------------------------------------- ---------------------------
  <span id="main-content" tabindex="-1"></span>
-------------------------------------------------- ---------------------------
<main role="main" id="main-content">
-------------------------------------------------- ---------------------------
<a href="https://schema.org/item" itemprop="item">
-------------------------------------------------- ---------------------------
<a href="https://schema.org/item" itemprop="item" aria-label="item"></a>
-------------------------------------------------- ---------------------------
LightHouse message:
Search engines may use href attributes on links to crawl websites. Ensure that the href attribute of anchor elements links to an appropriate destination, so more pages of the site can be discovered.
-------------------------------------------------- ---------------------------
Title of my post: https://earthtopomaps.com/earthtopomaps-deforestation-in-tierras-bajas-bolivia/
<a href="https://earthtopomaps.com/earthtopomaps-deforestation-in-tierras-bajas-bolivia/" itemprop="item">

**Comment by earthtopomaps** [comment]
I have e  problem with this: `<a href="" itemprop="item">` 
I have this problem since I changed the theme, I tried several scripts without good results.
I tried this, with no good results:

`<ol itemscope itemtype="https://schema.org/BreadcrumbList">`
`<li itemprop="itemListElement" itemscope 
       itemtype="https://schema.org/ListItem">`
`<a itemprop="item" href="https://example.com/dresses">`
`<span itemprop="name">Dresses</span></a>`
`<meta itemprop="position" content="1" />`
`</li>`
`<li itemprop="itemListElement" itemscope
      itemtype="https://schema.org/ListItem">`
`<a itemprop="item" href="https://example.com/dresses/real">`
`<span itemprop="name">Real Dresses</span></a>`
`<meta itemprop="position" content="2" />`
`</li>`
`</ol>`
I probably need to remove this from my theme: `itemprop="item"` 
but I don't know how to do it.
LigthtHouse Results:
[https://googlechrome.github.io/lighthouse/viewer/?gist=6ce87442a903ff6b970be9bbefc41993](url)
![Links are not crawlable](https://github.com/validator/validator/assets/22265545/22bbc1d3-df1e-41de-8fe8-49c55bcb9f9b)
Note: I don't have this problem with my home page. Only with my posts.

**Comment by sideshowbarker** [comment]
Added some additional messages.
